Web/Admin.UI/src/app/components/sub-pre-qual-report/sub-pre-qual-report-subnav/sub-pre-qual-report-subnav.component.html

```
@@ -0,0 +1,10 @@
+<div class="sub-pre-qual-report-subnav">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)